spec/staking.md

```diff
@@ -0,0 +1,13 @@
+# Staking
```

Comment:
Is staking based per lending pool, or throughout the entire entropy protocol?

Reply:
Right now staking will be separate from lending. We need to rewrite lending to use it as a staking pool.